project/Dependencies.scala

```diff
@@ -4,7 +4,7 @@ import Keys._
 object Dependencies {
 
   val AkkaVersion = "2.5.19"
-  val AkkaHttpVersion = "10.0.13" // TODO #281 update to 10.1.x when play and lagom are released
+  val AkkaHttpVersion = "10.0.13" // FIXME API #281 update to 10.1.x when play and lagom are released
```

Comment:
@renatocaval Do you know if it's ok if we bump this to 10.1.x now, when releasing Akka management 0.50 or later? New Play and Lagom releases should be ok and we will release new reactive-lib for 0.50.

Reply:
Yes, that's ok for 0.50.

Reply:
This was done in 
https://github.com/akka/akka-management/pull/427
also needed for JDK 11